tox.ini

```diff
@@ -11,4 +11,12 @@ deps =
     littleutils
 passenv = 
     FIX_EXECUTING_TESTS
+    ADD_EXECUTING_TESTS
     EXECUTING_SLOW_TESTS
+
+[testenv:py311]
+deps =
+    asttokens
+    pytest
+    littleutils
+    rich
```

Comment:
I should actually make these test/extra requirements within setup.cfg like my other projects, rather than testenv deps. But in the meantime, please just add `rich` to `.github/workflows/test.yml` so that build failures do the correct thing.

Reply:
rich is only available for python >3.6. Is there a way to install it only for 3.11?